src/FSharpx.Collections.Experimental/SkewBinomialHeap.fs

```diff
@@ -0,0 +1,307 @@
+﻿namespace FSharpx.Collections.Experimental
+
+//TODO: This should be better documented
+
+open FSharpx.Collections
+open System.Collections.Generic
+
+exception IncompatibleMerge of string
+
+type private 'T SBHTree = Node of 'T * 'T list * 'T SBHTree list
+
+type private 'T SBHTreeRoot = Root of int * 'T SBHTree
+
+[<AutoOpen>]
+module private Helper =
+    let inline xor a b = (a && not b) || (not a && b)
```

Comment:
let xor a b = (a && not b) || (not a && b)

        xor true true
        xor true false
        xor false true
        xor false false

        true <> true
        true <> false
        false <> true
        false <> false


==> just replace all occurences of xor with `<>`